src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int MAXIMUM_NUMBER_OF_BALL = 100;
+    private static Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public static int randomNumber(int number) {
+        int randomnumber = random.nextInt(number);
+        return randomnumber;
+    }
```

Comment:
I don't think we need this method here, lets just use Random where we need https://medium.com/att-israel/should-you-avoid-using-static-ae4b58ca1de5